src/operator/subgraph/tensorrt/onnx_to_tensorrt.cc

```diff
@@ -18,7 +18,7 @@
  */
 
 /*!
- * Copyright (c) 2019 by Contributors
+ * Copyright (c) 2020 by Contributors
  * \file onnx_to_tensorrt.cc
  * \brief TensorRT integration with the MXNet executor
  * \author Marek Kolodziej, Clement Fuji Tsang
```

Comment:
If you are changing the copyright years then add yourself to the authors too ;-)

Reply:
+1